docs/06-entity-&-operations.md

```diff
@@ -0,0 +1,28 @@
+# 6. Entity & Operations
+
+## 6.1 Legal Structure
+
+*Coming soon*
+
+## 6.2 Finances
+
+Protocol Guild's legal entity currently receives 2% of split proceeds to Safe multisigs:
+
+- Mainnet
+    - [0x0cDF1a78f00f56ba879D0aCc0FDa1789e415f23B](https://app.safe.global/balances?safe=eth:0x0cDF1a78f00f56ba879D0aCc0FDa1789e415f23B) (current address)
+    - [0x69f4b27882eD6dc39E820acFc08C3d14f8e98a99](https://app.safe.global/balances?safe=eth:0x69f4b27882eD6dc39E820acFc08C3d14f8e98a99) (previous address)
+- Optimism
+    - [0x0cDF1a78f00f56ba879D0aCc0FDa1789e415f23B](https://app.safe.global/balances?safe=oeth:0x0cDF1a78f00f56ba879D0aCc0FDa1789e415f23B)
+
+The entity converts all tokens on a monthly basis to maintain a stablecoin reserve. This covers:
+
+- $200k in reserve for 2 years of fixed + variable costs incurred to operate the entity
+- $100k in reserve for potential insurance claims against the foundation 
+- Stable base compensation for independent Operations contributors
+    - $12.5k/month ($150k/year) to support cheeky-gorilla's ongoing full-time contributions
+
+Generally, the entity should receive sufficient funding to cover the above, but not accumulate beyond that. These amounts are set by the membership, as noted in the [Governance](https://protocol-guild.readthedocs.io/en/latest/02-membership.html#governance) subsection of the docs.
```

Comment:
The term 'generally' is at odds with the rest of the entire statement? should it just be removed? or are we wanting to say these amounts are not fixed and may be larger, even though they're set by the membership, because that'd be what we're saying by having 'generally' here...